src/main/java/uk/gov/hmcts/reform/sendletter/entity/Letter.java

```diff
@@ -69,12 +61,58 @@ public Letter(
         this.additionalData = additionalData;
         this.type = type;
         this.pdf = pdf;
-        this.sentToPrintAt = null;
-        this.printedAt = null;
         this.isFailed = false;
     }
 
     public UUID getId() {
         return id;
     }
+
+    public String getMessageId() {
+        return messageId;
+    }
+
+    public String getService() {
+        return service;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public LetterState getState() {
+        return state;
+    }
+
+    public void setState(LetterState state) {
+        this.state = state;
+    }
+
+    public byte[] getPdf() {
+        return pdf;
+    }
```

Comment:
missing setter?

Reply:
I didn't put in a setter since we don't currently need to update PDFs. The pdf is a required parameter when you create a Letter. If we need to update the PDF in future we could add a setter then.